### DISCUSSION TITLE
[OP Stack Deployment]Connecting OP stack rollup to local devnet

### DISCUSSION THREAD

**Discussion by Gokulakrishnan-Rajan** [ethereum-optimism/developers#526] [Deploying the OP Stack]
### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

Tryin to test l2 rollup on local environment following the documentation
issue: I am trying to create L2 rollup in the local environment and try to connect it to the hardhat l1 node(or a devnet) and test the l2 network.

I need to run the network and test the functionalities which requires test eths. to bypass this issue I need to have my local l1 network to connect and get test tokens.

### Logs

CRIT [07-10|15:56:24.421] Application failed message="failed to setup: unable to create the rollup node: failed to init the runtime config: failed to load runtime configuration repeatedly, last error: operation failed permanently after 5 attempts: failed to fetch unsafe block signing address from system config: invalid value: string "0x5e5cd7041609c08c9e7c98597de25bd101b475c71e88f9928d3a9da01de60b07", expected a 8 byte hex string at line 1 column 221"

### Additional Information

Kindly let me know if there is a way to create a op L2 rollup and connect it to the Local l1 network or devnet where I can test multiple functions . 
I have started the hardhat node, op-geth, both are running without any errors. I am getting the above mentioned error when I try to start the node by following 
./bin/op-node \
  --l2=http://localhost:8551 \
  --l2.jwt-secret=./jwt.txt \
  --sequencer.enabled \
  --sequencer.l1-confs=5 \
  --verifier.l1-confs=4 \
  --rollup.config=./rollup.json \
  --rpc.addr=0.0.0.0 \
  --rpc.port=8547 \
  --p2p.disable \
  --rpc.enable-admin \
  --p2p.sequencer.key=$GS_SEQUENCER_PRIVATE_KEY \
  --l1=$L1_RPC_URL \
  --l1.rpckind=$L1_RPC_KIND
  --l1.trustrpc

**Comment by opfocus** [comment]
The above prompt is 8 bytes. Maybe you can try to remove the 0x of $GS_SEQUENCER_PRIVATE_KEY.
In addition, if you need devnet, you can refer to this document section, which includes L1+L2 and the latest chain features.
https://docs.optimism.io/chain/testing/dev-node
